net/socket/Kconfig

```diff
@@ -5,13 +5,12 @@
 
 menu "Socket Support"
 
-config NET_NACTIVESOCKETS
-	int "Max socket operations"
-	default 16 if !DEFAULT_SMALL
-	default 4 if DEFAULT_SMALL
+config NET_NACTIVESOCKETS_PER_BLOCK
+	int "Max socket operations per block"
+	default 4
 	---help---
-		Maximum number of concurrent socket operations (recv, send,
-		connection monitoring, etc.). Default: 16
+		Maximum number of concurrent socket operations per block
```

Comment:
The number of concurrent socket operations per allocation

Reply:
Done